lib/Backend/arm/LowerMD.cpp

```diff
@@ -3154,7 +3154,7 @@ LowererMD::GenerateFastBrOrCmString(IR::Instr* instr)
         if (instr->GetDst()->IsInt32())
         {
             opndSuccess = IR::IntConstOpnd::New(!isCmNegOp ? 1 : 0, TyMachReg, this->m_func);
-            opndFailure = IR::IntConstOpnd::New(!isCmNegOp ? 0 : 0, TyMachReg, this->m_func);
```

Comment:
This looks good. I couldn't find the corresponding code for int32 dst in LowerMDShared.cpp. Do you know why that is?